site/src/pages/release-notes/1.6.0.mdx

```diff
@@ -0,0 +1,315 @@
+  <type://HealthCheckService> anymore. #3382
+  - You should specify <type://TransientServiceOption#WITH_TRACING> if you want to trace them.
+  ```java
+  HealthCheckService.builder()
+                    .transientServiceOptions(WITH_TRACING)
+                    ...
+                    .build();
+  ```
+- You can now clean up resources by overriding <type://FilteredStreamMessage#onCancellation(Subscriber)>
+  when `Subscription.cancel()` is called. #3375
+  ```java
+  new MyFilteredHttpResponse(res) {
+      ...
+      @Override
+      protected void onCancellation(Subscriber<? super U> subscriber) {
+          // Clean up resources.
+      }
+  }
+  ```
+- You can now easily set 'cookie' or 'set-cookie' headers. #3391
+  ```java
+  Cookie cookie = Cookie.of("cookie", "value");
+  RequestHeaders headers = RequestHeaders.builder(HttpMethod.GET, "/")
+                                         .cookie(cookie)
+                                         .build();
+
+  Cookie setCookie1 = ...
+  Cookie setCookie2 = ...
+  ResponseHeaders headers = ResponseHeaders.builder(HttpStatus.OK)
+                                           .cookies(setCookie1, setCookie2)
+                                           .build();
+  ```
+- You can now use `req.root_id` <type://BuiltInProperty> to log the id of <type://RequestContext#root()>.
+  #3429 #3433
+- You can now use Thrift 0.14.0. #3470 #3422
+- You can now use OAuth 2.0 related features. #2268 #2840
+  - [OAuth 2.0 server-side token authorization](https://datatracker.ietf.org/doc/rfc7662/) using
+    <type://OAuth2TokenIntrospectionAuthorizer>
+  - [Client credential grants](https://datatracker.ietf.org/doc/html/rfc6749#section-4.4) using
+    <type://OAuth2ClientCredentialsGrant>
+  - [Resource Owner Password Credentials Grant](https://datatracker.ietf.org/doc/html/rfc6749#section-4.3)
+    using <type://OAuth2ResourceOwnerPasswordCredentialsGrant>
+  - [OAuth 2.0 Token Revocation](https://datatracker.ietf.org/doc/rfc7009/) using <type://TokenRevocation>
+
+  <Tip>
+
+    OAuth 2.0 features are currently experimental, so you should be careful using the feature.
+
+  </Tip>
+
+- You can now use [Jakarta RESTful Web Services](https://jakarta.ee/specifications/restful-ws/)
+  on top of Armeria with the `armeria-resteasy` module. #3285 #3296
+
+  <Tip>
+
+    `armeria-resteasy` is currently experimental, so you should be careful using the feature.
+
+  </Tip>
+
+## 📈 Improvements
+
+- Various improvements for documentation. #3237 #3289 #3361 #3390 #3401 #3427
+
+## 🛠️ Bug fixes
+
+- You no longer see `NullPointerException` in <type://HttpResponseDecoder>. #3036 #3407
+- You no longer see 405 when the exact and param path are defined with different HTTP methods. #3330 #3340
+- You no longer see `Address family not supported by protocol` or `Connection refused` error anymore. #3425
+- You no longer see escaped Unicode from a JSON response in <type://DocService>. #3396
```

Comment:
The Unicode characters like emojis in a JSON response are now rendered correctly in DocService. ?